components/layout.tsx

```diff
@@ -20,258 +23,41 @@ export const Layout: React.FC<React.PropsWithChildren<LayoutProps>> = ({
-                        href="/why-use-maple/for-individuals"
-                        handleClick={closeNav}
-                      >
-                        {t("navigation.forIndividuals")}
-                      </NavLink>
-                    </NavDropdown.Item>
-                    <NavDropdown.Item>
-                      <NavLink
-                        href="/why-use-maple/for-orgs"
-                        handleClick={closeNav}
-                      >
-                        {t("navigation.forOrganizations")}
-                      </NavLink>
-                    </NavDropdown.Item>
-                    <NavDropdown.Item>
-                      <NavLink
-                        href="/why-use-maple/for-legislators"
-                        handleClick={closeNav}
-                      >
-                        {t("navigation.forLegislators")}
-                      </NavLink>
-                    </NavDropdown.Item>
-                  </NavDropdown>
-
-                  {authenticated && (
-                    <NavLink
-                      className={"navLink-primary"}
-                      handleClick={() => {
-                        closeNav()
-                        void signOutAndRedirectToHome()
-                      }}
-                    >
-                      {t("signOut", { ns: "auth" })}
-                    </NavLink>
-                  )}
-                </Nav>
-              </Navbar.Collapse>
-            </Navbar>
-          </NavBarBox>
-          <div
-            className={
-              isMobile
-                ? "me-2 w-100 h-100 flex"
-                : `col d-flex justify-content-start align-items-center`
-            }
-          >
-            <div
-              className={
-                isMobile
-                  ? `position-absolute top-0 start-50 z-1 translate-middle-x`
-                  : ""
-              }
-            >
-              <Nav.Link href="/" className="py-0 px-2 w-100">
-                {isMobile ? (
-                  <Image
-                    src="/Logo2024.png"
-                    alt="logo"
-                    className="w-100"
-                    width="60"
-                    height="60"
-                  />
-                ) : (
-                  <Image
-                    src="/Logo2024.png"
-                    alt="logo"
-                    width="80"
-                    height="80"
-                  />
-                )}
-              </Nav.Link>
-            </div>
-          </div>
-          <NavBarBox className={`justify-content-end`}>
-            <ProfileLink
-              role={claims?.role}
-              fullName={profile?.fullName}
-              isMobile={isMobile}
+    <>
+      {isClient ? (
+        <>
+          <Head>
+            <title>{formattedTitle}</title>
+            <link rel="icon" href="/favicon.ico" />
+          </Head>
+          <PageContainer>
+            {isMobile ? <MobileNav /> : <DesktopNav />}
```

Comment:
My one big concern for this PR - after seeing the updated Figma, do we still feel like we need two separate nav components for mobile and desktop, or do you think we can make it work with just the idiomatic bootstrap nav components now that the Log in / Sign Up button isn't split out?

I know we're planning to move closer to the Figma in the next PR, but this feels like a more structural question worth addressing before building on these further.

Reply:
before the refactor, the Navbar was more consistent across views with only a few minor changes and it made since to have one unifed component.

under the current design, the links are rearranged and reorganized depending on the view width, it no longer makes sense to have one component try to do two arrangements.

Reply:
I don't want to push too much harder here (and don't consider this a blocker), but in the design linked in the PR description, I'm seeing that the link categories/orders look identical across desktop and mobile. Am I looking at the right part of the Figma (the "Navigation V2" bit)?

Reply:
I've made an alternative unified navbar.

The pros are all the navbar concerns are in one component.

The cons are that the readability suffers.